docs/upgrade.html

```diff
@@ -59,6 +59,9 @@ <h4><a id="upgrade_3_2_0" href="#upgrade_3_2_0">Upgrading to 3.2.0 from any vers
         (or to take advantage of <a href="#upgrade_11_exactly_once_semantics">exactly once semantics</a>),
         the newer Java clients must be used.
     </li>
+    <li>Upgrade from Scala 2.12.15 to 2.12.16. See https://github.com/scala/scala/releases/tag/v2.12.16 for release
```

Comment:
Is this comment at the right place? Currently, it seems like that it is under the section "For a rolling upgrade" which contains instructions for Kafka cluster administrators  to handle version upgrades which should have nothing to do with scala version since scala is bundled as a dependency independent of the scala version on the OS.

@ijuma will know more about this.

Reply:
We don't usually add patch upgrades to the release notes.

Reply:
I removed the these release and forced pushed the branch. The PR description contains the release notes instead.